doc/devices/ahs_local.rst

```diff
@@ -0,0 +1,105 @@
+The local AHS device
+====================
+
+The local AHS device of the PennyLane-Braket plugin runs Analogue Hamiltonian Simulation on the local Braket SDK. This
+could be either utilizing the processors of your own PC, or those of a `Braket notebook instance <https://docs.aws.amazon.com/braket/latest/developerguide/braket-get-started-create-notebook.html>`_ hosted on AWS.
+
+This device is useful for small-scale simulations in which the time of sending a job to a remote service would add
+an unnecessary overhead. It can also be used for rapid prototyping before running a computation
+on a paid-for remote service.
+
+
+Usage
+~~~~~
+
+After the Braket SDK and the plugin are installed you immediately have access to the `local Braket AHS device <https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html#braket-simulator-ahs-local>`_ in PennyLane.
```

Comment:
Should we mention that `jax` and `jaxlib` (`0.4.3`) need to be installed?

Reply:
Added!

Reply:
So technically for the local and remote ahs devices we shouldnt need jax, since jax is only required for the numerical ode solver. So potentially we should change when and where that import error is raised in `ParametrizedEvolution` 🤔